Modules/Services/Sources/Models/Details/FileDetails.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+public struct FileDetails {
+    
+    public var name: String { objectDetails.name }
+    public var id: String { objectDetails.id }
+    
+    private let objectDetails: ObjectDetails
+    
+    init(objectDetails: ObjectDetails) {
+        self.objectDetails = objectDetails
+    }
+}
```

Comment:
why we should save objectDetails here instead of setting `name` and `id` in init?

Reply:
fixed